core/blocks.js

```diff
@@ -16,11 +16,18 @@
 goog.module('Blockly.blocks');
 
 
+/**
+ * A block definition.  For now this very lose, but it can potentially
```

Comment:
this very lose -> this is very loose

Reply:
Oh bother.  GitHub didn't update the page to show the comment, only the approval.  🙄